packages/deploy-tooling/src/cli/index.ts

```diff
@@ -60,6 +60,7 @@ export function createCommand(name: 'deploy' | 'undeploy'): Command {
         command
             .option('--package <abap-package>', 'Package name for deploy target ABAP system')
             .option('--description <description>', 'Project description of the app')
+            .option('--create-transport <description>', 'Create transport request during deployment')
```

Comment:
I'm wondering if we need to change the behaviour;

if `--create-transport` is passed as a cli param, then a new TR is generated whether or not the config passed in contains the bookmark `CREATE_TR_DURING_DEPLOY`.